bindings/src/main/scala/chrome/pageAction/PageAction.scala

```diff
@@ -13,10 +13,10 @@ object PageAction {
 
   val onClicked: EventSource[Tab] = bindings.PageAction.onClicked
 
-  def show(tabId: Tab.Id): Unit =
+  def show(tabId: js.UndefOr[Int] = js.undefined): Unit =
```

Comment:
Is there any reason to replace the `Tab.Id` by the `Int`?

Reply:
I was matching this interface to both the `BrowserApi.scala` interface and to what is returned by `chrome.tabs.Tabs.query`. If you prefer to use the Tab.Id stuff, that's fine, its easy enough to unbox

Reply:
That would be ideal, let's try to keep layers above bindings Scala idiomatic.